### PR TITLE
Streamline docs navagation

### PR DESCRIPTION
**Pull request by vexr**
This proposed PR includes many changes aimed at streamlining navigation and improving the appearance of the category and guide names for the documents. Added redirects to any impacted links.

- Redirect from the `Farm from Anywhere` landing page directly to the guides. For now, a simple redirect is implemented, but a more permanent solution can be utilized if approved. If the landing page is retained, it should be completely reworked, as it currently fails to fully outline the contents of the guides.

- Allow users to close the top header message: `Learn more about the Autonomys Network by visiting our Academy.`

- Automatically expand the `Farming & Staking` category as it is the most popular section.
- Renamed the following categories and guide names.
  - Category `Space Acres (Recommended)` to `✨ Space Acres`
  - Category `SubWallet (Recommended)` to `✨ SubWallet`
  - Category `Operators and Nominators` to `Operators & Nominators` to fit on a single line.
  - Guide `Introduction to Staking and Operators` to `Getting Started`
  - Guide `Space Acres UI Translation Guide` to `Translation Guide` as its already in the Space Acres category.
- Removed category description pages for:
  - Space Acres
  - Advanced CLI
  - Farming & Staking - (also removed unused farming-&staking/index.md) 
- Reordered
  - Farming - According to expected popularity
  - Advanced CLI - According to expected popularity
  - Farming & Staking - Alphabetically since there is no specific step-by-step progression among all the subcategories. 
- Rewrite
  - `Farming` Category description to remove reference to testnet and tSSC.

### CODE REVIEW

File: docs/farming-&-staking/farming/_category_.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
     "label": "Farming",
-    "position": 3,
+    "position": 1,
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/_category_.json:3]: It's a bit weird that Wallets come after farming, I'd suggest keeping the order:

Wallets, Farming, Operators & Nominators, Timekeeping

File: docs/farming-&-staking/farming/space-acres/_category_.json [hunk 2]
```diff
@@ -1,8 +1,5 @@
 {
-    "label": "Space Acres (Recommended)",
-    "position": 2,
-    "link": {
-      "type": "generated-index",
-      "description": "Autonomys GUI Application Space Acres"
-    }
+    "label": "✨ Space Acres",
+    "position": 3,
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/_category_.json:3]: Similarly inside the farming category - I'd suggest keeping the order: Getting Started, Space Acres, Advanced CLI, Common Problems, Additional Guides

File: docs/farming-&-staking/farming/space-acres/translate_space_acres.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Space Acres UI Translation Guide
+title: Translation Guide
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/translate_space_acres.md:2]: Makes sense 👍

File: docs/farming-&-staking/farming/space-acres/_category_.json [hunk 4]
```diff
@@ -1,8 +1,5 @@
 {
-    "label": "Space Acres (Recommended)",
-    "position": 2,
-    "link": {
-      "type": "generated-index",
-      "description": "Autonomys GUI Application Space Acres"
-    }
+    "label": "✨ Space Acres",
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/_category_.json:2]: Unfortunately, ✨ meaning might not be clear for some users, I think we have to keep **(recommended)** for both the SubWallet and Space Acres. 

I like the idea though

File: docs/farming-&-staking/wallets/subwallet.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
-title: SubWallet (Recommended)
+title: SubWallet
+label: "✨ SubWallet"
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/wallets/subwallet.md:3]: Similarly to Space Acres, let's please keep the (Recommended) label

**Review by EmilFattakhov — CHANGES_REQUESTED** [review]
Thanks very much @vexr, I think it's a great idea to get rid of the index pages and just direct users straight to the documentation. And you're right about the homepage - we still likely want to have one, but it needs to be drastically re-designed and updated. 

Requested a few small changes

File: docs/farming-&-staking/farming/space-acres/_category_.json (hunk 4 above)
Comment (vexr) [reply docs/farming-&-staking/farming/space-acres/_category_.json:2]: Can you give me your thoughts on this new approach: Added a floating label below the sidebar to indicate recommended applications. This should better clarify the purpose of the stars without needing to use `(recommended)` in the navigation. This also allows for future applications and guides to be easily recommended in a cleaner fashion.

**Comment by vexr** [comment]
> I think it's a great idea to get rid of the index pages

Do you think it's worth keeping them for collections of more than four guides, or is there no need for any of them?

File: docs/farming-&-staking/farming/space-acres/_category_.json (hunk 4 above)
Comment (EmilFattakhov) [reply docs/farming-&-staking/farming/space-acres/_category_.json:2]: I still think you have to know where to look to see the label explaining what the ✨ means. 
I don't mind giving this approach a try and seeing if there's any user feedback about it.

**Comment by EmilFattakhov** [comment]
> I reverted to the previous hero landing page. While it doesn't need to be addressed immediately, having an appealing landing page that simultaneously showcases the documentation navigation and provides direct access to the guides would be beneficial. Currently, it feels a bit like a compliance feature for accessing the documentation, especially for returning users

Sorry I might have not been clear, I think we should go ahead with your idea to remove the landing page and only turn it back on once we've updated the design. At the moment, your suggestion to take users directly to the Quick Start page provides a much better onboarding experience.

**Comment by EmilFattakhov** [comment]
> Do you think it's worth keeping them for collections of more than four guides, or is there no need for any of them?

Yes, let's please keep it for large categories, where it improves usability/readability.

**Review by EmilFattakhov — APPROVED** [review]
Excellent work, thank you very much!

**Pull request merged**